docs/getting_started/notifications.md

```diff
@@ -6,8 +6,10 @@ Jellyfish sends server side notifications whenever some important event occurs,
 * component has crashed
 * etc.
 
+Notifications can be received through websockets or webhooks.
+
 A full list of available notifications is always present in specific server SDK documentation.
-Here, we present how you can subscribe for server side notifications using Elixir Server SDK:
+Here, we present how you can subscribe for server side websocket notifications using Elixir Server SDK:
```

Comment:
Let's create two subsections: Websocket and Webhooks. It will be easier to navigate 
A sentence or two describing what are ws and what are webhook notifications would also be appreciated (in their specific sections)